### PR TITLE
[16.0][MIG] carrier_account_environment

### PR DESCRIPTION
**Pull request by florian-dacosta**
Standart migration

### CODE REVIEW

File: carrier_account_environment/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,30 @@
+With this module installed, the delivery carrier are
+configured in the `server_environment_files` module (which is a module
```
Comment (hparfr) [inline carrier_account_environment/readme/CONFIGURE.rst:2]: can you also talk about the alternatives like [server_environment_data_encryption](https://github.com/OCA/server-env/tree/14.0/server_environment_data_encryption)

**Review by hparfr — CHANGES_REQUESTED** [review]

**Review by rousseldenis — APPROVED** [review]
Code review

**Review by bguillot — APPROVED** [review]

**Comment by florian-dacosta** [comment]
@hparfr If you are ok with the README change, can you merge the PR please?

**Review by hparfr — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**